js/views/components/ModeratorCard.js

```diff
@@ -36,6 +36,8 @@ export default class extends BaseVw {
     if (!this.model || !(this.model instanceof Profile)) {
       throw new Error('Please provide a Profile model.');
     }
+
+    handleLinks(this.el);
```

Comment:
If you're going to be stopProgogation'ing on an element, you would want to pass that element into handleLinks so any links will still be handled.

This could be on any element. Doesn't need to be the main view element. It just happens to be slightly more convenient if you are stopping propagation on the main view element, because then you don't have to call handleLinks() in render.